src/test/regress/expected/columnar_drop.out

```diff
@@ -12,35 +12,39 @@
 -- 'postgres' directory is excluded from comparison to have the same result.
 -- store postgres database oid
 SELECT oid postgres_oid FROM pg_database WHERE datname = 'postgres' \gset
-SELECT count(distinct storage_id) AS columnar_stripes_before_drop FROM columnar.stripe \gset
 -- DROP columnar tables
-DROP TABLE contestant;
-DROP TABLE contestant_compressed;
+SET client_min_messages TO WARNING;
+DROP TABLE IF EXISTS contestant;
+DROP TABLE IF EXISTS contestant_compressed;
```

Comment:
I feel like this is making the test quite useless on the second run, because it's not actually dropping anything anymore. I feel like it would be better to:
1. create new tables 
2. check they have stripes
3. drop them
4. check that there's no stripes anymore

Or another option would be to simply mark this test as non-repeatable.